src/index.js

```diff
@@ -57,22 +78,13 @@ function route(url, replace=false) {
 
 /** Check if the given URL can be handled by any router instances. */
 function canRoute(url) {
-	for (let i=ROUTERS.length; i--; ) {
-		if (ROUTERS[i].canRoute(url)) return true;
-	}
-	return false;
+	return ROUTERS.some(router => router.canRoute(url));
 }
 
 
 /** Tell all router instances to handle the given URL.  */
 function routeTo(url) {
-	let didRoute = false;
-	for (let i=0; i<ROUTERS.length; i++) {
-		if (ROUTERS[i].routeTo(url)===true) {
-			didRoute = true;
-		}
-	}
-	return didRoute;
+	return ROUTERS.reduce((didRoute, router) => (router.routeTo(url) === true || didRoute), false);
```

Comment:
Sort of: I cocked-up putting pull request #136 on master, and brought them in here with poor git skills.  I can resubmit.

Please check pull request #136 again: these methods (`some` and `reduce`) have been supported since way back in IE9, so should be fine...  No polyfill bloat required.